File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/test/java/simulacionBandaMusica/factories/BajistaFactoryTest.java
package simulacionBandaMusica.factories;

import org.junit.jupiter.api.Test;
import simulacionBandaMusica.models.Bajista;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class BajistaFactoryTest {

    @Test
    void createBajistaTest() {
        Bajista Bajista = BajistaFactory.create();
        // Bajista no nulo, para ver si se han asignado correctamente los datos
        assertNotNull(Bajista);
    }
}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/test/kotlin/simulacionBandaMusica/factories/BajistaFactoryTest.kt
package simulacionBandaMusica.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import simulacionBandaMusica.models.Musico.Bajista

internal class BajistaFactoryTest {
    @Test
    fun createBajistaTest() {
        val Bajista: Bajista = BajistaFactory.create()
        // Bajista no nulo, para ver si se han asignado correctamente los datos
        Assertions.assertNotNull(Bajista)
    }
}